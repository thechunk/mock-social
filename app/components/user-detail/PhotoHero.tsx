import * as React from 'react';
import {PureComponent} from 'react';
import AlbumContainer from '../common/album-container/AlbumContainer';
import styles from './styles';
import {BlankImageUri} from "../../styles/global";
import LoadedImage from "../common/image/LoadedImage";

export default class PhotoHero extends PureComponent<IPhotoHero> {
    render() {
        return (
            <AlbumContainer
                userId={this.props.userId}
                limit={1}
                renderItem={v => <LoadedImage
                    style={styles.photoHero}
                    source={{uri: v.url}}
                    defaultSource={{uri: BlankImageUri}} />} />
        )
    }
}
