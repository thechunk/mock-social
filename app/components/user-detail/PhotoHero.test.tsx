import * as React from 'react';
import {shallow} from 'enzyme';
import PhotoHero from './PhotoHero';

describe('PhotoHero', () => {
    it('renders correctly', () => {
        expect(shallow(<PhotoHero photos={[]} />)).toMatchSnapshot();
    });
});