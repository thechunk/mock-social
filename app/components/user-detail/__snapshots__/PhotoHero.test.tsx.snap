// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoHero renders correctly 1`] = `
<AlbumContainer
  limit={1}
  renderItem={[Function]}
/>
`;
